README.md

```diff
@@ -11,7 +11,7 @@
 - Stylelint (A mighty, modern linter that helps you avoid errors and enforce conventions in your styles).
 - To get a local copy up and running follow these simple example steps
 
-## Live Demo (if available)
+## Live Demo
 
```

Comment:
https://github.com/Regiss05/To-do-list/blob/ebe115a1d4890cd0e5928d48ecf0b9d7c3148cef/src/index.js#L1-L2

- [ ] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practices. Please Instead of disabling `eslint`, it will be nice to fix all linter errors manually.